app/src/main/java/woowacourse/shopping/di/DIContainer.kt

```diff
@@ -0,0 +1,54 @@
+package woowacourse.shopping.di
+
+import woowacourse.shopping.data.repository.CartDefaultRepository
+import woowacourse.shopping.data.repository.CartRepository
+import woowacourse.shopping.data.repository.ProductDefaultRepository
+import woowacourse.shopping.data.repository.ProductRepository
+import kotlin.reflect.KClass
+import kotlin.reflect.full.primaryConstructor
+
+class DIContainer {
+    private val instances = mutableMapOf<String, Any>()
```

Comment:
```suggestion
    private val instances = mutableMapOf<KClass<*>, Any>()
```
이렇게 하게 되면 nullable인 simpleName을 안쓰게 되어서 !!를 사용하지 않아도 될 것 같네요😊